src/windows/functions.cpp

```diff
@@ -393,6 +393,12 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterValid(TCHAR driveLetter) {
+  int driveLetterIndex = static_cast<int>(driveLetter) - static_cast<int>('A');
```

Comment:
I guess one way of avoiding these casts would be to switch the `while (logicalDrivesMask) {` loop to iterating by integer index instead of `currentDriveLetter`, and then only converting the integer index to an _actual_ driveLetter at the last possible moment?

Alternatively, maybe it'd be helpful if the `MountUtilsLog` functions actually specified _which_ `currentDriveLetter` they're trying to operate on?

Something I've just thought of - does this function still behave correctly when there are network shares mounted to drive letters?